classes/query.php

```diff
@@ -122,16 +122,61 @@ private function get_search_fields() {
         return array_values($searchfields);
     }
 
+    /**
+     * Add * wildcard to beginning and end of search string.
+     *
+     * @param string $q The query string.
+     * @return string $q The query string with added wildcards.
+     */
+    private function add_wildcards($q) {
+
+        $terms = explode(" ", $q); // Break search string into individual words.
+        $wildcardterms = array();
+
+        // Add wildcards to start and end of words.
+        foreach ($terms as $term) {
+
+            // Ignore words: and, or.
+            if ($term == 'and' || $term == 'or') {
```

Comment:
@mattporritt should we thing about case for this words? like AND Or And etc ?

Reply:
You're right, we should make it some thing like:
if (strtolower($term) == 'and' || strtolower($term) == 'or') {
We don't need to change the term in the query itself, Elasticsearch takes care of that for us.

Reply:
fixed